usvm-ts/src/test/kotlin/org/usvm/util/TSTestResolver.kt

```diff
@@ -10,32 +11,38 @@ import org.jacodb.ets.base.EtsRefType
 import org.jacodb.ets.base.EtsStringType
 import org.jacodb.ets.base.EtsType
 import org.jacodb.ets.base.EtsUndefinedType
+import org.jacodb.ets.base.EtsUnknownType
 import org.jacodb.ets.base.EtsVoidType
 import org.jacodb.ets.model.EtsMethod
+import org.jacodb.ets.model.EtsMethodParameter
+import org.usvm.TSContext
 import org.usvm.TSObject
+import org.usvm.TSRefTransformer
 import org.usvm.TSTest
+import org.usvm.UConcreteHeapRef
 import org.usvm.UExpr
 import org.usvm.USort
 import org.usvm.extractBool
 import org.usvm.extractDouble
 import org.usvm.extractInt
+import org.usvm.extractOrThis
 import org.usvm.memory.URegisterStackLValue
+import org.usvm.model.UModelBase
 import org.usvm.state.TSMethodResult
 import org.usvm.state.TSState
+import org.usvm.types.first
 
-class TSTestResolver {
+class TSTestResolver(
+    private val state: TSState
```

Comment:
Why? Resolver should be able to resolve many states independently

Reply:
```TSTestResolver``` was instantiated for each returned state, and I needed access to ```state.typeStorage```.
So I decided to add state as private constructor param.